packages/appeal-reply-service-api/globalConfig.json

```diff
@@ -0,0 +1 @@
+{"mongoUri":"mongodb://127.0.0.1:55215/jest?","mongoDBName":"jest"}
```

Comment:
Missing newline at end of file (should be done automatically via EditorConfig

Reply:
Done. Editor is installed and working, but not for this file, strangely.